hash/spectrum_cass.xml

```diff
@@ -141624,7 +142434,7 @@ Nothing happens after loading the cassette
 	</software>
 
 	<software name="theoryof_b" cloneof="theoryof">
```

Comment:
Not introduced with this PR, but the questions and exercises tapes shouldn’t be clones of `theoryof` since they’re a different piece of software.